Buildings/Controls/OBC/Utilities/PIDWithAutotuning/Validation/DirectActingPIDWithFirstOrderAMIGO.mo

```diff
@@ -0,0 +1,150 @@
+within Buildings.Controls.OBC.Utilities.PIDWithAutotuning.Validation;
+model DirectActingPIDWithFirstOrderAMIGO
```

Comment:
We may change the class name in the validation package as:
`DirectActingPID`, `DirectActingPI`, `ReverseActingPID`, `ReverseActingPI`.

Reply:
As suggested, I changed the names but keep <code>WithFirstOrderAMIGO</code> as other system models may be added in the future.